UI/js/common.js

```diff
@@ -0,0 +1,21 @@
+/*
+UI errors for user display
+1000 -> user errors
+*/
+const errors = Object.freeze({
+    //1000 user errors
+    1000: 'Invalid username or password',
```

Comment:
should we do 1xxx for system and 2xxx for users?

Reply:
Was thinking of that, sure I shall do that :)